internal/storage/activities/delete_amss.go

```diff
@@ -17,6 +17,8 @@ import (
 
 type DeleteFromAMSSLocationActivity struct {
 	ssclient *http.Client
```

Comment:
Is there some reason to have an `ssclient` field in this struct rather than just using a local variable in the `Execute()` method?  The `ssclient` value is overwritten with a new client every time `Execute()` is called, so I can't see any reason to persist the value outside of the `Execute()` scope.